packages/react-switch/src/components/Switch/useSwitch.ts

```diff
@@ -1,25 +1,41 @@
 import * as React from 'react';
+import { getNativeElementProps, useId, resolveShorthand } from '@fluentui/react-utilities';
 import { useSwitchState } from './useSwitchState';
-import { resolveShorthandProps, makeMergeProps } from '@fluentui/react-utilities';
-import type { SwitchProps, SwitchState } from './Switch.types';
+import type { SwitchProps, SwitchState, SwitchSlots } from './Switch.types';
```

Comment:
nit: Order these alphabetically

```suggestion
import type { SwitchProps, SwitchSlots, SwitchState } from './Switch.types';
```